pyomo/gdp/plugins/binary_multiplication.py

```diff
@@ -0,0 +1,173 @@
+from .gdp_to_mip_transformation import GDP_to_MIP_Transformation
```

Comment:
I know it's in contrib, but can you please add the copyright statement at the top?